### ISSUE TITLE
Implement __repr__ methods for CS and Model classes

### ISSUE THREAD

**Issue by padamson** [USNavalResearchLaboratory/nepc#11]
Addresses `__repr__` tasks for #7 

For CS, return "CS(cursor, cs_id)".

For Model, return "Model(cursor, model_name)".

**Comment by arichar6** [comment]
So, I don’t have much experience implementing `__repr__`. Shouldn’t it give an expression that would recreate the object? Eg “CS(cursor=..., cs_id=...)”, where the parameters needed to create the specific instance are provided explicitly?

**Comment by padamson** [comment]
Yes, I think you're right.  That's what I meant by my poorly written comment.  I looked at it briefly and couldn't see an easy way to get the cursor info. Perhaps we need a repr for nepc_connect and call it?

**Comment by padamson** [comment]
This should wait until we move the CustomCS logic into CS. (See #12.)  Then the `__repr__` will just return the command required to build the CS object "from scratch" by providing data and metadata.

There might be a way to have `__repr__` return the cursor string required to create the CS from the database, but it's not obvious to me after looking at the MySQL Connector/Python documentation.
